diff -up CoinUtils-2.9.7/Makefile.in.orig CoinUtils-2.9.7/Makefile.in
--- CoinUtils-2.9.7/Makefile.in.orig	2013-11-01 12:32:02.953485683 -0200
+++ CoinUtils-2.9.7/Makefile.in	2013-11-01 12:32:37.098486990 -0200
@@ -336,7 +336,7 @@ EXTRA_DIST = $(am__append_2)
 ########################################################################
 pkgconfiglibdir = $(libdir)/pkgconfig
 pkgconfiglib_DATA = coinutils.pc
-addlibsdir = $(DESTDIR)$(datadir)/coin/doc/CoinUtils
+addlibsdir = $(DESTDIR)$(datadir)/doc/coin-or-$(PACKAGE_NAME)
 
 ########################################################################
 #                         Maintainer Stuff                             #
@@ -346,7 +346,7 @@ CLEANFILES =
 # Files that are generated and should be cleaned with make distclean
 DISTCLEANFILES = $(am__append_3) $(VPATH_DISTCLEANFILES)
 DocFiles = README AUTHORS LICENSE
-DocInstallDir = $(datadir)/coin/doc/$(PACKAGE_NAME)
+DocInstallDir = $(datadir)/doc/coin-or-$(PACKAGE_NAME)
 all: all-recursive
 
 .SUFFIXES:
