diff -up CoinUtils-2.9.7/src/Makefile.in.orig CoinUtils-2.9.7/src/Makefile.in
--- CoinUtils-2.9.7/src/Makefile.in.orig	2013-11-01 12:33:35.313489219 -0200
+++ CoinUtils-2.9.7/src/Makefile.in	2013-11-01 12:33:54.010489935 -0200
@@ -64,7 +64,7 @@ am__strip_dir = `echo $$p | sed -e 's|^.
 am__installdirs = "$(DESTDIR)$(libdir)" "$(DESTDIR)$(includecoindir)"
 libLTLIBRARIES_INSTALL = $(INSTALL)
 LTLIBRARIES = $(lib_LTLIBRARIES)
-libCoinUtils_la_LIBADD =
+libCoinUtils_la_LIBADD = -lbz2 -lz
 am_libCoinUtils_la_OBJECTS = CoinAlloc.lo CoinBuild.lo \
 	CoinDenseVector.lo CoinError.lo CoinFactorization1.lo \
 	CoinFactorization2.lo CoinFactorization3.lo \
