diff -up CoinUtils-2.8.7/CoinUtils/Makefile.in.orig CoinUtils-2.8.7/CoinUtils/Makefile.in
--- CoinUtils-2.8.7/CoinUtils/Makefile.in.orig	2012-09-27 21:20:53.187931622 -0300
+++ CoinUtils-2.8.7/CoinUtils/Makefile.in	2012-09-27 21:20:57.280932710 -0300
@@ -336,7 +336,7 @@ EXTRA_DIST = $(am__append_2)
 ########################################################################
 pkgconfiglibdir = $(libdir)/pkgconfig
 pkgconfiglib_DATA = coinutils.pc
-addlibsdir = $(DESTDIR)$(datadir)/coin/doc/CoinUtils
+addlibsdir = $(DESTDIR)$(datadir)/doc/coin-or-$(PACKAGE_NAME)-$(PACKAGE_VERSION)
 
 ########################################################################
 #                         Maintainer Stuff                             #
@@ -346,7 +346,7 @@ CLEANFILES =
 # Files that are generated and should be cleaned with make distclean
 DISTCLEANFILES = $(am__append_3) $(VPATH_DISTCLEANFILES)
 DocFiles = README AUTHORS LICENSE
-DocInstallDir = $(datadir)/coin/doc/$(PACKAGE_NAME)
+DocInstallDir = $(datadir)/doc/coin-or-$(PACKAGE_NAME)-$(PACKAGE_VERSION)
 all: all-recursive
 
 .SUFFIXES:
